db/setup/schema.sql

```diff
@@ -1,24 +1,24 @@
 CREATE TABLE
-  IF NOT EXISTS LocalFile(
-    "id" UUID DEFAULT gen_random_uuid() PRIMARY KEY,
+  IF NOT EXISTS LocalFile (
+    "id" UUID DEFAULT gen_random_uuid () PRIMARY KEY,
```

Comment:
Sorry for these formatting changes. I've installed the [Prettier SQL](https://marketplace.visualstudio.com/items?itemName=inferrinizzard.prettier-sql-vscode) extension to format SQL files, which uses the [sql-formatter](https://github.com/sql-formatter-org/sql-formatter) library.

Reply:
I think is a good idea, I'll install it to!